src/components/molecules/CameraMicrophoneControls/CameraMicrophoneControls.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useCallback } from "react";
+import classNames from "classnames";
+import Video, {
+  AudioTrackPublication,
+  VideoTrackPublication,
+} from "twilio-video";
+
+import { DEFAULT_CAMERA_ENABLED } from "../../../settings";
```

Comment:
```suggestion
import { DEFAULT_CAMERA_ENABLED } from "settings";
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2221/commits/34c2da4fc00fb2f75d093deb88233a7c5846ec92